inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/ConfigConstants.java

```diff
@@ -73,5 +73,6 @@ public class ConfigConstants {
 
     public static String HTTP = "http://";
     public static String HTTPS = "https://";
+    public static int MAX_MESSAGE_LENGTH = 500 * 1024;
```

Comment:
Manager already supports configuring the maximum packet length maxPacketLength setting allowed by DataProxy cluster for each {groupId, streamId} [1]. 

SDK needs to check the length of the reported message based on the maxPacketLength provided by Manager.

1. https://github.com/apache/inlong/pull/11314

Reply:
I have finished it in #11119